pkg/generators/forwarding/fluentd/output_fluentd_buffer.go

```diff
@@ -27,19 +27,30 @@ const (
 )
 
 func (olc *outputLabelConf) ChunkLimitSize() string {
-	if hasBufferConfig(olc.forwarder) {
+	if hasBufferConfig(olc.forwarder) && olc.forwarder.Fluentd.Buffer.ChunkLimitSize != "" {
 		return string(olc.forwarder.Fluentd.Buffer.ChunkLimitSize)
-	}
+	} else {
+		size := ""
+		switch olc.Target.Type {
+		case loggingv1.OutputTypeFluentdForward:
+			size = "1m"
+		case loggingv1.OutputTypeElasticsearch, loggingv1.OutputTypeKafka, loggingv1.OutputTypeSyslog:
+			size = "8m"
+		default:
+			// This should not normally happen.
+			panic(fmt.Sprintf("unknown or mismatched type: %s", olc.Target.Type))
```

Comment:
Why the panic? Why can't the default be every thing but forward?

Reply:
I was not sure whether or not '8m' as default chunk_limit_size is suitable or not.
But it looks like forward is only an exceptional case and the others including Loki take '8m'. 
So I will change this part to "everything but forward is '8m' as default value" as you suggested.